mythril/analysis/report.py

```diff
@@ -113,16 +136,45 @@ def as_json(self):
 
     def as_swc_standard_format(self):
```

Comment:
should the format be named jsonv2 or swc_standard_format

Reply:
It's not really a swc standard so I guess `jsonv2` makes more sense.